Lab 1.2.1/Lab 1.2.1/LibraryUser.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Lab_1._2._1
+{
+    public interface ILibraryUser
+    {
+        void AddBook(string bookName);
+        void RemoveBook(string bookName);
+        string GetBookInfo(int index);
+        int GetBooksCount();
+    }
+
+    public class LibraryUser : ILibraryUser
+    {
+        private static int nextId;
+        public static int GetNextId()
+        {
+            return nextId++;
+        }
+
+        public string Phone { get; set; }
+        public int BookCount { get => this.bookList.Count; }
+
+        public readonly string FirstName;
+        public readonly string LastName;
+        public readonly int Id;
+        public readonly int BookLimit;
+        public readonly bool IsAnonymous;
+
+        private List<string> bookList;
```

Comment:
this is hack, you have to use array.
this is need in order to see how usefull will be other .Net classes if compare with build-in types

Reply:
Commit 8ed68811cec989030e8cd9b00e0e6001307a2b27